addons/web/tooling/_eslintignore

```diff
@@ -68,4 +68,9 @@ web_map/static/src/legacy/**/*
 web_map/static/tests/legacy/**/*
 
 # whitelist base_setup
-!base_setup/static/src/**/*
+!base_setup
+!base_setup/**/*
```

Comment:
Can you check that this works? Seems like it should be `addons/base_setup` in both cases.

Reply:
we're changing it right now